README.md

```diff
@@ -175,38 +183,48 @@ $ java -jar cromwell.jar run my_wf.wdl - - my_wf.metadata.json
 ... play-by-play output ...
 $ cat my_wf.metadata.json
 {
+  "workflowName": "w",
   "calls": {
-    "example.my_task": [{
+    "w.x": [{
       "executionStatus": "Done",
```

Comment:
oh, perhaps it's not very valuable then! We could remove that field (full disclosure: I only added it because I thought it'd be handy for my personal debugging)